### PR TITLE
chore: improve clarity and consistency of doc descriptions

### PR DESCRIPTION
**Pull request by kylecarbs**
Ran this through ChatGPT with a few iterations to improve the clarity of each description.

Soon our docs will have an `og:image` and a social image auto-generated for each page:

image

### CODE REVIEW

File: docs/manifest.json [hunk 1]
```diff
@@ -3,268 +3,270 @@
...
+          "description": "Learn how to audit commands in workspaces with exectrace.",
           "path": "./templates/process-logging.md",
           "state": "enterprise"
         },
         {
           "title": "Icons",
-          "description": "Coder includes icons for popular cloud providers and programming languages for you to use",
+          "description": "Learn about the icons C
+
+oder includes for popular cloud providers and programming languages.",
```
Comment (stirby) [inline docs/manifest.json:269]: What's this?

File: docs/manifest.json [hunk 2]
```diff
@@ -277,266 +279,269 @@
     },
     {
       "title": "IDEs",
-      "description": "Learn how to use your IDE of choice with Coder",
+      "description": "Learn how to use your IDE of choice with Coder.",
```
Comment (stirby) [inline docs/manifest.json:282]: I don't think we should use periods here.

File: docs/manifest.json [hunk 3]
```diff
@@ -1068,72 +1073,72 @@
...
       "title": "FAQs",
-      "description": "Frequently asked questions",
+      "description": "Learn about frequently asked questions.",
       "path": "./faqs.md",
       "icon_path": "./images/icons/info.svg"
     },
     {
       "title": "Guides",
-      "description": "Employee-authored tutorials",
+      "description": "Learn from employee-authored tutorials.",
```
Comment (stirby) [inline docs/manifest.json:1095]: It seems like most of these just add "Learn from" or "Learn about" which just elongates the descriptions?

File: docs/manifest.json (hunk 1 above)
Comment (kylecarbs) [reply docs/manifest.json:269]: Awkward

File: docs/manifest.json (hunk 3 above)
Comment (kylecarbs) [reply docs/manifest.json:1095]: These are meta tags used for SEO. Would you rather "Learn from" be prefixed in the Google description? I certainly think it adds clarity.

File: docs/manifest.json (hunk 1 above)
Comment (kylecarbs) [reply docs/manifest.json]: Fixed

**Pull request closed without merging**